### PR TITLE
Update Installation for OpenSUSE

### PR DESCRIPTION
**Pull request by BeardedTek**
OpenSUSE uses /etc/zypp/repos.d not /etc/yum.repos.d

Changes reflect as noted above.

None

## Description

As noted above

## Release notes

Notes:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +98,28 @@ sudo dnf install github-desktop
...
+```
+sudo rpm --import https://rpm.packages.shiftkey.dev/gpg.key
+sudo sh -c 'echo -e "[shiftkey-packages]\nname=GitHub Desktop\nbaseurl=https://rpm.packages.shiftkey.dev/rpm/\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\ngpgkey=https://rpm.packages.shiftkey.dev/gpg.key" > /etc/zypp/repos.d/shiftkey-packages.repo'
+```
+
+#### [@mwt](https://github.com/mwt) package feed
+
+```sh
+sudo rpm --import https://mirror.mwt.me/shiftkey-desktop/gpgkey
+sudo sh -c 'echo -e "[mwt-packages]\nname=GitHub Desktop\nbaseurl=https://mirror.mwt.me/shiftkey-desktop/rpm\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\ngpgkey=https://mirror.mwt.me/shiftkey-desktop/gpgkey" > /etc/zypprepos.d/mwt-packages.repo'
```
Comment (shiftkey) [inline README.md:114]: :art: I think we missed a slash here?

```suggestion
sudo sh -c 'echo -e "[mwt-packages]\nname=GitHub Desktop\nbaseurl=https://mirror.mwt.me/shiftkey-desktop/rpm\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\ngpgkey=https://mirror.mwt.me/shiftkey-desktop/gpgkey" > /etc/zypp/repos.d/mwt-packages.repo'
```

File: README.md [hunk 2]
```diff
@@ -82,6 +82,7 @@ sudo rpm --import https://mirror.mwt.me/shiftkey-desktop/gpgkey
 sudo sh -c 'echo -e "[mwt-packages]\nname=GitHub Desktop\nbaseurl=https://mirror.mwt.me/shiftkey-desktop/rpm\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\ngpgkey=https://mirror.mwt.me/shiftkey-desktop/gpgkey" > /etc/yum.repos.d/mwt-packages.repo'
 ```
 
+
```
Comment (shiftkey) [inline README.md:85]: :art: drop whitespace

```suggestion
```

**Review by shiftkey — COMMENTED** [review]
Thanks for the submission! I think there's one more fix we need to do here...

**Comment by BeardedTek** [comment]
Whoops, I think I hit the wrong thing there.  I'll be able to sit down later tonight to make sure I didn't mess up.

Thank you for the awesome utility!  It's a must have in my processes.  I'm glad I can contribute and make it easier for someone else!

**Comment by shiftkey** [comment]
Closing this out in favour of #864

**Pull request closed without merging**